### PR TITLE
Increase Marketplace image size to 5GB

### PR DESCRIPTION
**Pull request by aditjha-msft**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

This PR increases the Max Disk Size for the marketplace mariner image to 5GB by the request of Chris Co for the continuation of ADE VM extension work with Mariner.

###### Change Log  

Changed `MaxSize: 1500` in `marketplace-gen2.json` to `MaxSize: 5000` to represent increase to 5GB disk size.

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Pipeline build id: 268321
- Booted VHD in Hyper-V and command `df -H` showed that increase was implemented.

### CODE REVIEW

File: toolkit/imageconfigs/marketplace-gen2.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "Disks": [
         {
             "PartitionTableType": "gpt",
-            "MaxSize": 1500,
+            "MaxSize": 5000,
```
Comment (PawelWMS) [inline toolkit/imageconfigs/marketplace-gen2.json:5]: Is that the only image we want to change? What about the gen1 and ARM64 variants?

**Review by PawelWMS — COMMENTED** [review]
This seems like quite a big size increase. Should we also update baselines for any BVT tests checking for size regressions?

**Review by SeanDougherty — APPROVED** [review]

File: toolkit/imageconfigs/marketplace-gen2.json (hunk 1 above)
Comment (aditjha-msft) [reply toolkit/imageconfigs/marketplace-gen2.json:5]: I was told by Chris that this is only needed for the marketplace image that supports ADE extension work.

**Comment by aditjha-msft** [comment]
> This seems like quite a big size increase. Should we also update baselines for any BVT tests checking for size regressions?

@PawelWMS

**Comment by aditjha-msft** [comment]
> This seems like quite a big size increase. Should we also update baselines for any BVT tests checking for size regressions?

Talked with Tom Kennard, no BVT tests impacted with this because it is not affecting core images.

**Pull request merged**